src/audio.cpp

```diff
@@ -55,6 +55,13 @@ void APU::test()
 	printf("APU test\n");
 }
 
+// initialize the writehandler of memorymap
+void APU::initializeWriteHandler()
+{
+	if (mMap)
+		mMap->setAduioWriteHandler([this](Word address) { this->onWrite(address); });
```

Comment:
spelling mistake here

Reply:
give more descriptive name to `onWrite ()`

Reply:
How about `onMemoryWrite()`